tasks/main.yml

```diff
@@ -122,6 +131,26 @@
   with_items:
     - pacemaker
     - corosync
+  when: not cluster_is_remote | bool
```

Comment:
- [ ] ansible reports error with double declaration of `when` in this task. `when` statement should be unified here
~~~
[WARNING]: While constructing a mapping from .../roles/ondrejhome.ha-cluster-
pacemaker/tasks/main.yml, line 126, column 3, found a duplicate dict key (when). Using last defined value only.
~~~

Reply:
Oops.

Reply:
Done